details.browser.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
So, this file needs to be reworked based on the recent ARC related changes by @DylanPiercey . Check the root folder for examples of how other modules do this. You also need to add an entry to `browser.json` at the root.